### ISSUE TITLE
Sketchup::Model::get_product_family

### ISSUE THREAD

**Issue by DanRathbun** [SketchUp/ruby-api-stubs#36]
Should `Sketchup::Model::get_product_family` be marked as deprecated ?

I have a report that SketchUp 2015 Make is reporting `2` (pro) as a result of calling this method.

I remember reporting bugs in this method some versions back, and I thought that they were addressed. But perhaps still bugged in v2015 ? 

If so perhaps a note should be added to the docs about bugged versions ?

P.S. - FTR this method should not have been an instance method nor defined within the `Sketchup::Model` class.

**Comment by thomthom** [comment]
> P.S. - FTR this method should not have been an instance method nor defined within the Sketchup::Model class.
Yes, it is misplaced. This method is troublesome. Might be best to deprecate it and introduce a new one going forward.

**Comment by DanRathbun** [comment]
> @thomthom Yes, it is misplaced. This method is troublesome. Might be best to deprecate it and introduce a new one going forward.

**Agree** best choice. Such a important query method should have been within the **`Sketchup::Licensing`** module, [u]which should be frozen[/u].

When I reported the previous bugs, I suggested the old method be aliased to the new locked method.

Not that this issue will matter much going forward. ;)

**Comment by DanRathbun** [comment]
> @thomthom Might be best to deprecate it and introduce a new one going forward.

With the discontinuance of **Make** edition development (2018) the use of this old method is greatly reduced. This makes it a good time to deprecate the old method and create a new one.

New API Issue logged ...
* [Deprecate the Sketchup::Model#get_product_family instance method](https://github.com/SketchUp/api-issue-tracker/issues/85)
